canvas_modules/common-canvas/src/common-properties/controls/radioset/radioset.jsx

```diff
@@ -201,7 +201,7 @@ class RadiosetControl extends React.Component {
 				row: i
 			};
 			buttons.push(
-				<div key={i} className="properties-radioset-panel">
+				<div key={i} className={`properties-radioset-panel ${Number(this.props.control.values[i]) && "properties-radioset-panel-number"}`}>
```

Comment:
This is odd code.  Why does it matter if the value is a number or not?  This seems like a workaround more than a fix.

Reply:
Removed it and adjusted only css